src/Book.java

```diff
@@ -24,4 +25,21 @@ public Autor getAutor() {
     public void setPublicationDate(int publicationDate) {
         this.publicationDate = publicationDate;
     }
+    public String toString () {
+        return " Название книги " + this.title + " дата публикации " + this.publicationDate + " автор " + getAutor().getFirstname()+ " " + getAutor().getLastname();
```

Comment:
```suggestion
        return " Название книги " + this.title + " дата публикации " + this.publicationDate + " автор " + autor.toString();
```
не надо дублировать код toString класса Autor, надо просто вызвать метод и всё

Reply:
понял, спасибо